web/static/js/components/surveys/SurveyStatus.jsx

```diff
@@ -121,9 +131,11 @@ class SurveyStatus extends PureComponent {
           break
         } else {
           if (survey.nextScheduleTime) {
-            icon = 'access_time'
-            const date = new Date(survey.nextScheduleTime)
-            text = this.nextCallDescription(survey, date)
+            if (timezones && timezones.items) {
+              icon = 'access_time'
+              const date = new Date(survey.nextScheduleTime)
+              text = this.nextCallDescription(survey, date)
```

Comment:
does this cause a glitch/re-layout in the ui until the timezone is loaded or the content appears without changing the position of elements once the timezones list is available?